whitelabelco/sofo-foods
# Cp/feat/update brand copy

### Description 💻

Updates copy for Athena Farms brand

### Implementation 💡

Add Athena Farms content mentioned in this [issue](https://github.com/whitelabelco/sofo-foods/issues/158)
#### Type of change


### How to test 🔍

- `npm i`
- `npm run dev`
- navigate to `http://localhost:3000/`
- click "Brands"
- confirm Athena Farms content matches copy 

### Screenshots/Demos 📷

### Checklist ✅
